README.md

```diff
@@ -31,47 +31,43 @@ You can do this one of two ways:
 
 #### 1.1 Default Apps Script Developer Console Project
 
-1. Access the automatically created apps script Developer Console Project by
-  1. `Resources` > `Cloud Platform Project`
-  1. Click on the blue link at the top (`project-name - api-project-##########`) to access the correct Developer Console Project
+1. Go to the [Google Cloud Platform](https://console.cloud.google.com)
```

Comment:
Could you also refer to the other option that is by accesing `Resources` > `Cloud Platform Project` please?

Reply:
Sure, but can you explain from which point you can access `Resources`? I do not see this in the UI in https://console.developers.google.com 
This is where I got lost in the first place :-)

Reply:
@lricoy could you let me know how to proceed so I can update everything? Thanks! :-)

Reply:
@lricoy nevermind, discovered it myself! I added som more improvements and think it is ready!

Reply:
Hey @jceelen nice changes but I was referring to just entering `https://script.google.com/` and using the above menu of `Resources` > `Cloud Platform Project`.

Reply:
Hey @jceelen nice changes but I was referring to just entering https://script.google.com/ and using the above menu of Resources > Cloud Platform Project.